src/main/java/nl/tudelft/ti2206/group9/gui/popup/AbstractInfoPopup.java

```diff
@@ -97,7 +97,9 @@ public Button getRightButton() {
         return right;
     }
 
-    /** Plays the button sound once, given a volumelevel. */
+    /**
+     * Plays the button sound once, given a volumelevel.
+     */
```

Comment:
Besides, why did you change so many javadoc to the style above? I guess this has been autogenerated and it has been done for a lot of classes. :P 

<documentation